low-level/files/opt/seagate/sspl/setup/resource_map/server.py

```diff
@@ -19,17 +19,16 @@
 import time
 import re
 import errno
-from framework.utils.ipmi_client import IpmiFactory
-from framework.base import sspl_constants
-
-import errno
-import time
 import psutil
 from pathlib import Path
+
+from framework.utils.ipmi_client import IpmiFactory
+from framework.base import sspl_constants
 from framework.utils.tool_factory import ToolFactory
 from resource_map import ResourceMap
 from error import ResourceMapError
-from framework.base.sspl_constants import CPU_PATH
+from framework.base.sspl_constants import (CPU_PATH, SAS_RESOURCE_ID,
+                                           DEFAULT_RECOMMENDATION)
```

Comment:
```
from framework.base.sspl_constants import (CPU_PATH, SAS_RESOURCE_ID,
                                           DEFAULT_RECOMMENDATION)
```
This can be better in better style to avoid lines of character restriction later.
```
from framework.base.sspl_constants import (
     CPU_PATH, SAS_RESOURCE_ID, DEFAULT_RECOMMENDATION)
```

Reply:
addressed.